chapters/4-acquisition.tex

```diff
@@ -34,11 +34,11 @@
 
 \end{fullwidth}
 
-\section{Acquiring data from research partners}
+\section{Acquiring data ethically}
```

Comment:
In addition to "ethically", I see "transparent" and "ensuring reproducibility" as threads through the section. Would this title be too long?
```suggestion
\section{Acquiring data ethically and transparently while ensuring reproducibility }
```

Reply:
The suggestion is a bit long, i changed to 'acquiring data ethically and reproducibly', sound ok?

Reply:
yes, that sounds good!